xml/System.Data/DataRowView.xml

```diff
@@ -754,6 +763,10 @@
   
```

Comment:
line 752

Change it to "Column or relation name when getting, column name when setting."

Reply:
I changed it but from the discussion with Diego, we will keep the original string.